### PR TITLE
Fix: Countermeasures tooltip for latin languages

### PR DESCRIPTION
**Pull request by xezon**
This change includes armor buff information in latin tooltips of Countermeasures upgrade.

* Related to #1135

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 1]
```diff
@@ -48131,16 +48131,16 @@ AR: "ﺔﻠﺗﺎﻘﻤﻟا ﺢﺒﺸﻟا ةﺮﺋﺎﻃ ﺪﻳوﺰﺗ ﻢﺘ
 END
 
 CONTROLBAR:TooltipUSAUpgradeCountermeasures
-US: "All aircraft capable of evading 30% of missiles fired at them"
-DE: "Alle Flugzeuge können 30% der auf sie abgefeuerten Raketen ausweichen"
-FR: "Tous les avions peuvent éviter les missiles une fois sur deux avec des contre-mesures activées."
-ES: "Todos los aviones pueden evitar los misiles el 30% de las veces"
-IT: "Tutti gli aerei con Contromisure attive hanno il 30% di possibilità di evitare i missili."
+US: "All aircraft become capable of evading 30% of missiles fired at them and get 37% more armor against bullets"
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/generals.str:48134]: It's -25% for Spectre. They did the math right for that one.

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/generals.str:48134]: Also, 37.5% rounds to 38.

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str:48134]: Rounded to 38. As for Spectre, explaining that in tooltip would make it too complicated.

**Comment by xezon** [comment]
Looks okay.

**Pull request merged**